cli/http_util.rs

```diff
@@ -29,10 +33,12 @@ lazy_static! {
       USER_AGENT,
       format!("Deno/{}", version::DENO).parse().unwrap(),
     );
+    headers.insert(ACCEPT_ENCODING, HeaderValue::from_static("gzip, br"));
```

Comment:
Note: the same client is used for `fetch` API - make sure that it works as well.